### DISCUSSION TITLE
Add the ability to push and pull on a schedule?

### DISCUSSION THREAD

**Discussion by gudstrand** [Vinzent03/obsidian-git#275] [General]
I would love to see the ability to push and pull on a schedule. For example, it would be great to push all my changes from one PC at 5PM and have my other PC scheduled to pull changes at 5:30PM. Also to avoid conflicts it would be great to be able to set an active window for p[ushes and pulls. For example, you could have an interval of 15 minutes but have it only run within an 8Am to 5PM window.

**Comment by Vinzent03** [comment]
I turned this into an issue #276
